src/test/java/com/google/devtools/build/lib/syntax/MethodLibraryTest.java

```diff
@@ -486,6 +486,7 @@ public void testRange() throws Exception {
         .testStatement("str(range(5, 0, -1))", "[5, 4, 3, 2, 1]")
```

Comment:
Add a test for mult, which is also now disabled?

Reply:
I thought I had one... I guess I was wrong... :)